app/views/works/_work_header_navigation.html.erb

```diff
@@ -92,7 +92,7 @@
       <%= render 'subscriptions/form', :subscription => @subscription %>
     </li>
   <% end %>
-
+  <% if @work.posted?  %>
```

Comment:
The indenting here should be fixed -- everything after `<% if @work.posted? %>` (which has an extra space in it after the ?, by the way) and before the `<% end %>` should now move over two spaces.